Plugin example: module not found

### Search before asking

- [X] I searched the [issues](https://github.com/sqlfluff/sqlfluff/issues) and found no similar issues.


### What Happened

It seems some module changes were not applied properly in SQLFluff 1.4.1 (or earlier): in \plugins\sqlfluff-plugin-example\test\rules\rule_test_cases_test.py.
I see an error in this file: module "sqlfluff.testing.rules" is not found, it seems it was replaced with "sqlfluff.utils.testing.rules".

### Expected Behaviour

No errors

### Observed Behaviour

When running plugin tests for my plugin I receive an error:
```
..
from sqlfluff.testing.rules import load_test_cases, rules__test_helper
E   ModuleNotFoundError: No module named 'sqlfluff.testing.rules'
```

### How to reproduce

Run test for plugin rules in embedded plugin example

### Dialect

not applicable

### Version

SQLFluff 1.4.1 (or may be earlier, I didn't tested since 0.13.1), 
Python 3.9.13 

### Configuration

not applicable

### Are you willing to work on and submit a PR to address the issue?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/sqlfluff/sqlfluff/blob/main/CODE_OF_CONDUCT.md)

--- a/plugins/sqlfluff-plugin-example/test/rules/rule_test_cases_test.py
+++ b/plugins/sqlfluff-plugin-example/test/rules/rule_test_cases_test.py
@@ -1,7 +1,7 @@
 """Runs the rule test cases."""
 import os
 import pytest
-from sqlfluff.testing.rules import load_test_cases, rules__test_helper
+from sqlfluff.utils.testing.rules import load_test_cases, rules__test_helper
 
 ids, test_cases = load_test_cases(
     test_cases_path=os.path.join(

